src/gov/nist/core/LexerCore.java

```diff
@@ -736,7 +736,8 @@ public String number() throws ParseException {
 
         int startIdx = ptr;
         try {
-            if (!isDigit(lookAhead(0))) {
+            char c = lookAhead(0);
+            if (c != '-' && !isDigit(c)) {
```

Comment:
![image](https://user-images.githubusercontent.com/7847241/27135933-1ea66c34-511a-11e7-8aea-07a24da6daf3.png)
this a brief of main usages for "number" method including different types of headers